app/javascript/components/ActionItemCards/styles.js

```diff
@@ -0,0 +1,46 @@
+export const styles = () => ({
+  card: {
+    display: 'flex',
+    flexdirection: 'row',
+    justifycontent: 'start',
```

Comment:
to transfer the styles from css --> jss you have to change the kebab case to camelcase. So `justify-content` becomes `justifyContent` etc.